packages/protocol/contracts/bridge/EtherVault.sol

```diff
@@ -65,14 +65,30 @@ contract EtherVault is EssentialContract {
      *********************/
 
     /**
-     * Send Ether from EtherVault to the sender, checking they are authorized.
+     * Transfer Ether from EtherVault to the sender, checking that the sender
+     * is authorized.
      * @param amount Amount of ether to send.
      */
-    function receiveEther(uint256 amount) public onlyAuthorized nonReentrant {
+    function returnEther(uint256 amount) public onlyAuthorized nonReentrant {
```

Comment:
i dont think the naming `returnEther` matches with the functionality well? `transferEther` lines up with the EventName better?

Reply:
How about I change the Event name? `transferEther ` seems really common to me...